plugins/node/opentelemetry-instrumentation-mongodb/examples/package.json

```diff
@@ -38,7 +38,7 @@
     "@opentelemetry/instrumentation-mongodb": "^0.32.0",
     "@opentelemetry/sdk-trace-node": "^1.0.0",
     "@opentelemetry/sdk-trace-base": "^1.0.0",
-    "mongodb": "^3.7.3"
+    "mongodb": "^3.6.11"
```

Comment:
Yes, this was intentional at the time I made that commit at least. It was so that this range overlapped with the `"mongodb": "3.6.11",` in "plugins/node/opentelemetry-instrumentation-mongodb/package.json" one directory up.  At the time (earlier in the development of this PR) I had hit some npm install error.  I don't have the details of that install error anymore, sorry.

I could try to remove that change and re-test. (I might not get to that today.)

Reply:
No need to re-test, just tested myself as I was curious and it installs fine now with the previous range. 🙂 
I think we can keep it at what you have now. 👍